app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+def channels
```

Comment:
Would you fix indentation? 

Also do we need this function?

Reply:
Fixed, consolidated this to one function